tests/tests/optionsTests.robot

```diff
@@ -87,4 +83,21 @@ Find and open Collision resolution option
     List Selection Should Be    collisionResolutionOptions    error
     Wait Until Element Is Visible    ${ReloadBadge}
 
+Verify Output log to a file option
+    [Documentation]    Verify Output log to a file option
+    [Tags]    options
 
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${file_4mb}[archive_name].zip
+    Click Add Files Button
+    Ubuntu Upload Test File    ${file_4mb}[path]
+    Click Load Button
+    Check If Log Console Contains    Opening "${file_4mb}[name]"
+    Validate Output Description    ${file_4mb}[archive_name]
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${LogsButton}
+    Click Element    ${OutputLogsSwitch}
+    Wait Until Element Is Visible    ${DownloadLogsButton}
+    Click Load Button
+    Click Element    ${DownloadLogsButton}
+    Switch Window    new
+    Wait Until Element Is Visible     //pre[contains(text(),'info: Opening "file_4MB.exe"')]
```

Comment:
this locator could be in locators file and used here as variable

Reply:
done